### PR TITLE
TEN-1341 / 24.04.2 / Rename relevant NFS methods to specify v3 or v4

### PR DESCRIPTION
**Pull request by jordan-mcgillivray005**

### CODE REVIEW

File: test_cases/webui/shares/nfs/test_delete_new_nfs_share.py [hunk 1]
```diff
@@ -39,10 +39,15 @@ def tear_down_test(self, nfs_data):
...
-    def test_delete_new_nfs_share(self, nfs_data):
+    @allure.story('Delete NFSv3 Share')
+    def test_delete_new_nfsv3_share(self, nfs_data):
         """
-        This test verifies the NFS share can be deleted.
+        This test verifies the NFSv3 share can be deleted.
+
+        Test Steps:
+        1. Delete the NFSv3 share.
+        3. Navigate to the datasets page and verify that the share is still detached from the dataset.
```
Comment (markbainix) [inline test_cases/webui/shares/nfs/test_delete_new_nfs_share.py:49]: numbering off

File: test_cases/webui/shares/nfs/test_edit_nfs_share.py [hunk 2]
```diff
@@ -48,10 +48,29 @@ def tear_down_test(self, nfs_data):
...
+
+        Test Steps:
+        1. Edit the NFSv3 share and configure it with advanced options:
+            - Authorized network address
+            - Authorized ip address
+            - Read only
+            - Maproot user
+            - Maproot group
+            - Security type
+        3. Navigate to the datasets page and verify that the share is attached to the dataset.
```
Comment (markbainix) [inline test_cases/webui/shares/nfs/test_edit_nfs_share.py:64]: numbering off

File: test_cases/webui/shares/nfs/test_nfsv3_share_advanced_configurations.py [hunk 3]
```diff
@@ -106,17 +106,17 @@ def test_nfs_share_authorized_ip_address(self, nfs_advanced_config):
...
-        Summary: This test sets an NFS share to read only and verifies that the share can be mounted but
+        Summary: This test sets an NFSv3 share to read only and verifies that the share can be mounted but
         only read access is granted.
 
         Test Steps:
-        1. Create an NFS share and set it to read only.
+        1. Create an NFSv3 share and set it to read only.
         2. Verify share can mount and that only read access is granted.
-        4. Unmount the NFS share.
+        4. Unmount the NFSv3 share.
```
Comment (markbainix) [inline test_cases/webui/shares/nfs/test_nfsv3_share_advanced_configurations.py:119]: numbering off

**Review by markbainix — CHANGES_REQUESTED** [review]

File: test_cases/webui/shares/nfs/test_nfsv3_share_advanced_configurations.py (hunk 3 above)
Comment (jordan-mcgillivray005) [reply test_cases/webui/shares/nfs/test_nfsv3_share_advanced_configurations.py:119]: Fixed.

File: test_cases/webui/shares/nfs/test_delete_new_nfs_share.py (hunk 1 above)
Comment (jordan-mcgillivray005) [reply test_cases/webui/shares/nfs/test_delete_new_nfs_share.py:49]: Fixed.

**Review by markbainix — APPROVED** [review]

**Pull request merged**